nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-web/src/main/resources/OSGI-INF/navtree-contrib.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0"?>
 <component name="org.nuxeo.ecm.virtualnavigation.tagsContrib">
 
+  <require>org.nuxeo.ecm.platform.actions</require>
```

Comment:
Is this really needed?

Reply:
Maybe not, I'll try without

Reply:
Looks like the contributed action is not correctly merged if this require is not added.